cli/src/pcluster/templates/cdk_builder_utils.py

```diff
@@ -872,6 +872,14 @@ def _build_policy(self) -> List[iam.PolicyStatement]:
                     )
                 ],
             ),
+            iam.PolicyStatement(
+                sid="CloudFormation",
+                actions=[
+                    "cloudformation:SignalResource",
```

Comment:
this action is correct for cfn-signal, but since cfn-signal is the wrong approach you should instead replace it with the permissions required by the complete-lifecycle-action operation.

Reply:
Done!